packages/clarity-js/src/data/variable.ts

```diff
@@ -0,0 +1,35 @@
+import { Constant, Event, VariableData } from "@clarity-types/data";
+import encode from "./encode";
+
+export let data: VariableData = null;
+
+export function set(variable: string, value: string): void {
```

Comment:
What happens if the same variable is set multiple times? What does that mean if it happens?

Reply:
Since these are variables, the assumption is that the value will be overridden and the last value will prevail.